src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -343,6 +366,17 @@ private function hasProductWithName(array $products, string $name): bool
         return false;
     }
 
+    private function hasProductWithOriginalPrice(array $products, int $originalPrice): bool
+    {
+        foreach ($products as $product) {
```

Comment:
Why do we have an array of products here? Also, this loop just checks if there is any product with `original_price`, not the particular, which seems bad